plugins/bd-xray.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: bd-xray
+spec:
+  version: v0.1.1
+  platforms:
+    - selector:
+        matchLabels:
+          os: linux
+          arch: amd64
+      uri: "https://github.com/blackducksoftware/kubectl-bd-xray/releases/download/v0.1.1/kubectl-bd-xray_v0.1.1_linux_amd64.tar.gz"
+      sha256: e2dde42fa702f3fbd5015d42315645dedb665b88af975425d2dec8747f743108
+      files:
+        - from: "bd-xray"
+          to: "."
+        - from: "LICENSE"
+          to: "."
+      bin: "bd-xray"
+    - selector:
+        matchLabels:
+          os: darwin
+          arch: amd64
+      uri: "https://github.com/blackducksoftware/kubectl-bd-xray/releases/download/v0.1.1/kubectl-bd-xray_v0.1.1_darwin_amd64.tar.gz"
+      sha256: 6a04074eb61c6b759e32444b054db27f6477113ac231ce5f3bcff2c8ab905531
+      files:
+        - from: "bd-xray"
+          to: "."
+        - from: "LICENSE"
+          to: "."
+      bin: "bd-xray"
+    - selector:
+        matchLabels:
+          os: windows
+          arch: amd64
+      uri: "https://github.com/blackducksoftware/kubectl-bd-xray/releases/download/v0.1.1/kubectl-bd-xray_v0.1.1_windows_amd64.zip"
+      sha256: f454b9c5e84ee1b004b072cbca0b3ff982d1578a45efab4cbe798bc97d06dece
+      files:
+        - from: "bd-xray.exe"
+          to: "."
+        - from: "LICENSE"
+          to: "."
+      bin: "bd-xray.exe"
+  shortDescription: Run Black Duck Image Scans for software composition analysis.
+  homepage: https://github.com/blackducksoftware/kubectl-bd-xray
+  description: |
+    This plugin runs Black Duck Image Scans for software composition analysis.
+    Just point and scan images in any namespace, third-party or your own yaml files, helm charts and more.
```

Comment:
If you can line wrap at 80 chars, it would better fit the terminals.

Reply:
done, thanks!